tensorflow/compiler/xla/tests/convolution_test.cc

```diff
@@ -408,7 +408,9 @@ class Convolve1D_1x2x5_1x2x2_WithPadding : public ConvolutionTest {
 TYPED_TEST_CASE(Convolve1D_1x2x5_1x2x2_WithPadding, TestTypes);
 TYPED_TEST(Convolve1D_1x2x5_1x2x2_WithPadding, Types) { this->RunTest(); }
 
-XLA_TEST_F(ConvolutionTest, Convolve3D_1x4x2x3x3_2x2x2x3x3_Valid) {
+// 5D tensors are not yet supported in ROCm
+XLA_TEST_F(ConvolutionTest,
+           DISABLED_ON_GPU_ROCM(Convolve3D_1x4x2x3x3_2x2x2x3x3_Valid)) {
```

Comment:
Please add a comment why these tests are not good for ROCm at present. Bonus points for linking to a Github issue ;-)

Reply:
will do.

Reply:
after reviewing with @deven-amd and @jerryyin we actually already have 5D tensor / 3D convolution support in ROCm. reverting changes here.